src/programs/print_stack_statistics/print_stack_statistics.cpp

```diff
@@ -46,13 +46,13 @@ bool PrintStackStatistics::DoCalculation()
 	Image my_image_one;
 	float image_sigma;
 	float image_average;
-	float average_sigma = 0.0f;
-	float average_average = 0.0f;
+	double average_sigma = 0.0f;
```

Comment:
I guess this is explicitly creating a float then casting it to double...

Reply:
fixed